src/_includes/docs/swift-package-manager/migrate-objective-c-plugin.md

```diff
@@ -51,7 +51,7 @@ The example below uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
            // TODO: Update the platforms your plugin supports.
            // If your plugin only supports iOS, remove `.macOS(...)`.
            // If your plugin only supports macOS, remove `.iOS(...)`.
-           .iOS("12.0"),
+           .iOS("13.0"),
```

Comment:
Do we need to bump this version? Plugins can keep supporting 12.0 if they'd like to support older Flutter SDK versions.

Reply:
I thought that the website always documented the minimums enforced by the tool on `stable` branch. Anyone on `stable` will have their project auto-migrated to 13.0. Any plugin that uses Flutter `stable` will _need_ to target 13.0 or greater since that's the new Flutter framework minimum.

Perhaps there's something about Swift Package Manager that I'm misunderstanding? If so, we should probably add a new variable for Swift Package Manager minimum iOS version and use that in these files.